TAF_EasyRest/TestFramework/Pages/SignUpPage.cs

```diff
@@ -0,0 +1,112 @@
+
+        public SignUpPage(IWebDriver driver) : base(driver)
+        {
+            NavigationMenu = new NavigationMenuPageComponent(driver);
+        }
+
+        private IWebElement _nameField => driver.FindElement(By.XPath("//input [@name='name']"));
+        private IWebElement _emailField => driver.FindElement(By.XPath("//input [@name='email']"));
+        private IWebElement _phoneNumberField => driver.FindElement(By.XPath("//input [@name='phoneNumber']"));
+        private IWebElement _birthDateField => driver.FindElement(By.XPath("//input [@name='birthDate']"));
+        private IWebElement _passwordField => driver.FindElement(By.XPath("//input [@name='password']"));
+        private IWebElement _confirmPasswordField => driver.FindElement(By.XPath("//input [@name='repeated_password']"));
+        private IWebElement _createAccountButton => driver.FindElement(By.XPath("//button[@type='submit']"));
+
+        public SignUpPage ClickNameField()
+        {
+            _nameField.Click();
+            return this;
+        }
+
+        public SignUpPage SendKeysToNameField(string name)
+        {
+            _nameField.SendKeys(name);
+            return this;
+        }
+
+        public SignUpPage ClickEmailField()
+        {
+            _emailField.Click();
+            return this;
+        }
+
+        public SignUpPage SendKeysToEmailField(string email)
+        {
+            _emailField.SendKeys(email);
+            return this;
+        }
+
+        public SignUpPage ClickPhoneNumberField()
+        {
+            _phoneNumberField.Click();
+            return this;
+        }
+
+        public SignUpPage SendKeysToPhoneNumberField(string phoneNumber)
+        {
+            _phoneNumberField.SendKeys(phoneNumber);
+            return this;
+        }
+
+        public SignUpPage ClickBirthDateField()
+        {
+            _birthDateField.Click();
+            return this;
+        }
+
+        public SignUpPage ChooseCurrentDate()
+        {
+            _birthDateField.Click();
+            driver.FindElement(By.XPath("//span[text()='OK']")).Click();
+            return this;
+        }
+
+        public SignUpPage ClickPasswordField()
+        {
+            _passwordField.Click();
+            return this;
+        }
+
+        public SignUpPage SendKeysToPasswordField(string password)
+        {
+            _passwordField.SendKeys(password);
+            return this;
+        }
+
+        public SignUpPage ClickConfirmPasswordField()
+        {
+            _confirmPasswordField.Click();
+            return this;
+        }
+
+        public SignUpPage SendKeysToConfirmPasswordField(string password)
+        {
+            _confirmPasswordField.SendKeys(password);
+            return this;
+        }
+
+        public SignUpPage ClickCreateAccountButton()
+        {
+            _createAccountButton.Click();
+            return this;
+        }
+
+        public override void GoToUrl()
```

Comment:
You probably created a branch not based on main. Because your BasePage class is different from the similar class of the main branch.
In this case, in your code, the GoToUrl method does not find where to overload.

Reply:
![image](https://user-images.githubusercontent.com/14205615/180826308-80a2cb35-002f-4c74-bf78-2a9d5477e418.png)